From: Andrew Morton <akpm@linux-foundation.org>
Subject: decode_stacktrace-decode-caller-address-checkpatch-fixes
Date: Tue Mar 10 11:48:47 AM PDT 2026

WARNING: 'substract' may be misspelled - perhaps 'subtract'?
#106: FILE: scripts/decode_stacktrace.sh:198:
+	# symbol, substract one from that to point the call instruction.
 	          ^^^^^^^^^

Please run checkpatch prior to sending patches

Cc: Masami Hiramatsu (Google) <mhiramat@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 scripts/decode_stacktrace.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/scripts/decode_stacktrace.sh~decode_stacktrace-decode-caller-address-checkpatch-fixes
+++ a/scripts/decode_stacktrace.sh
@@ -195,7 +195,7 @@ parse_symbol() {
 	# Evaluate it to find the actual address
 	# The stack trace shows the return address, which is the next
 	# instruction after the actual call, so as long as it's in the same
-	# symbol, substract one from that to point the call instruction.
+	# symbol, subtract one from that to point the call instruction.
 	if [[ $decode_retaddr == false && $offset != 0 ]]; then
 		expr=$((expr-1))
 	else
_
